rsconnect/actions.py

```diff
@@ -1523,14 +1529,26 @@ def create_notebook_deployment_bundle(
 
     if app_mode == AppModes.STATIC:
         try:
-            return make_notebook_html_bundle(file_name, python, image, hide_all_input, hide_tagged_input, None)
+            return make_notebook_html_bundle(
+                file_name,
+                python,
+                hide_all_input,
+                hide_tagged_input,
+                image=image,
+                check_output=None,
```

Comment:
We normally should not pass a value for `check_output`. That's a dependency injection point for test code to use.